File: crud/src/main/java/fr/cicco/crud/entity/CommandeMenu.java
package fr.cicco.crud.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Builder;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Entity
@Builder
public class CommandeMenu {
    @Id()
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Integer nbMenus;
    @ManyToOne
    @JoinColumn(name = "menu_id")
    private Menu menu;
    @ManyToOne
    @JoinColumn(name = "commande_id")
    private Commande commande;

}

File: crud/springCLI/python_entities_template/commandemenu.py

from ..datas.autocompletion import *

name = 'CommandeMenu'

entity = {
    
    'CommandeMenu' : {
            
        'id':{
            Type: Long,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'nbMenus':{
            Type: Integer,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'menu':{
            Type: "Menu",
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'commande':{
            Type: "Commande",
            options:{
                'dto':[
                ]
        
            }
        },

        }

}


